draft-regext-rdap-extensions.md

```diff
@@ -386,125 +484,187 @@ that example could be written as:
-"camel casing", in reference to the hump of a camel. In this style, the 
-first letter of every word, except the first word, composing a name is capitalized.
-This convention was adopted to visually separate the namespace from the
-name, with an underscore between them.
-
-Though there is no explicit guidance to use camel case names, extensions would
-be wise to continue the style.
-
-# Two Classes of Extensions {#extension_classes}
-
-Though all RDAP extensions are to be registered in the IANA RDAP extensions
-registry, there is an implicit two-class system of extensions that comes from
-the inherit ownership of the RDAP specifications by the IETF: extensions
-created by the IETF and extensions not created by the IETF.
-
-In the perspective of how extensions identifiers are used as namespace
-separators, extensions created by the IETF are not required to be prefixed
-with an extension identifier as the IETF can coordinate its own activities
-to avoid name collisions. In practice, most extensions owned by the IETF do use
-extension identifiers.
-
-RDAP extensions not defined by the IETF MUST use the extension identifier
-as a prefix in accordance with this document, [@!RFC7480], [@!RFC9082], and
-[@!RFC9083]. And RDAP extensions defined by the IETF SHOULD use the extension
-identifier as a prefix or as a bare extension identifier (see (#bare_extension)).
-IETF-defined RDAP extensions that do not follow this guidance MUST describe
-the necessity to do so.
+Usage of a bare extension identifier contravenes the guidance in
+[@!RFC9083].  This document updates [@!RFC9083] to explicitly allow
+this pattern.
+
+Along similar lines, an extension may define a single new object
+class, and use the extension's identifier as the object class name.
+
+### rdapConformance population
+
+[@!RFC9083, Section 4.1] offers the following guidance on including
+extension identifiers in the "rdapConformance" member of an RDAP
+response:
+
+    A response to a "help" request will include identifiers for all of
+    the specifications supported by the server. A response to any
+    other request will include only identifiers for the specifications
+    used in the construction of the response.
+
+A strict interpretation of this wording where "construction of the
+response" refers to the JSON structure only would rule out the use of
+(#profiles_and_markers) extension identifiers, which are in common use
+in RDAP.  This document updates the guidance. For responses to queries
+other than "/help", a response MUST include in the "rdapConformance"
+array only those extension identifiers necessary for a client to
+deserialize the JSON and understand the semantic meaning of the
+content within the JSON, and each extension identifier MUST be free
+from conflict with the other identifiers with respect to their syntax
+and semantics.
+
+Note that this document does not update the guidance from [@!RFC9083,
+Section 4.1] regarding "/help" responses and the "rdapConformance"
```

Comment:
[@!RFC9083, Section 4.1] needs to be on the same line to be properly de-referenced.

Reply:
Thanks, updated.